scripts/mysql/schema-0.5.0-mysql.sql

```diff
@@ -98,4 +98,22 @@ CREATE TABLE IF NOT EXISTS `fileset_version_info` (
     KEY `idx_mid` (`metalake_id`),
     KEY `idx_cid` (`catalog_id`),
     KEY `idx_sid` (`schema_id`)
-) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT 'fileset version info';
\ No newline at end of file
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_bin COMMENT 'fileset version info';
+
+CREATE TABLE IF NOT EXISTS `topic_meta` (
+    `topic_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'topic id',
+    `topic_name` VARCHAR(128) NOT NULL COMMENT 'topic name',
+    `metalake_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'metalake id',
+    `catalog_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'catalog id',
+    `schema_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'schema id',
+    `comment` VARCHAR(256) DEFAULT '' COMMENT 'topic comment',
+    `properties` MEDIUMTEXT DEFAULT NULL COMMENT 'topic properties',
```

Comment:
Will there be multiple version requirements for this property in the future?

Reply:
In my opinion, properties do not need multiple versions, while the schema of messages may require multiple versions, but we do not support message schema at this version.